railties/test/isolation/abstract_unit.rb

```diff
@@ -16,6 +16,9 @@
 require "active_support/testing/stream"
 require "active_support/testing/method_call_assertions"
 require "active_support/test_case"
+require "minitest/retry"
+
+Minitest::Retry.use!(verbose: false)
```

Comment:
Maybe we should only retry once? The default is 3.

Reply:
I assumed 3 was a good amount, but we can trial run this with one and increase as needed 👍